CodeChecker storage fails for certain files

We get the following fault on the server side when storing the result of xerces.
This makes the storage of the whole run to fail:

17:05:30] [29020] <139951674660608> - client_db_access_handler.py:1444 storeZip() - Parsing input file 'clangsa_TraverseSchema.cpp_9c8d802e.plist'
[17:05:30] [29020] <139951674660608> - plist_parser.py:132 parse_plist() - Parsing plist: /tmp/tmpVmkRWB/reports/clangsa_TraverseSchema.cpp_9c8d802e.plist
[ERROR] [17:05:30] - 'utf8' codec can't decode byte 0xb7 in position 301483: invalid start byte

--- a/libcodechecker/server/client_db_access_handler.py
+++ b/libcodechecker/server/client_db_access_handler.py
@@ -1487,7 +1487,8 @@ class ThriftRequestHandler(object):
                     LOG.debug(file_name + ' not found or already stored.')
                     continue
 
-                with codecs.open(zip_file_name, 'r', 'UTF-8') as source_file:
+                with codecs.open(zip_file_name, 'r',
+                                 'UTF-8', 'replace') as source_file:
                     file_content = source_file.read()
                     # TODO: we may not use the file content in the end
                     # depending on skippaths.

--- a/libcodechecker/server/client_db_access_server.py
+++ b/libcodechecker/server/client_db_access_server.py
@@ -222,6 +222,8 @@ class RequestHandler(SimpleHTTPRequestHandler):
             return
 
         except Exception as exn:
+            import traceback
+            traceback.print_exc()
             LOG.error(str(exn))
             self.send_error(404, "Request failed.")
             return

